### PR TITLE
JSON fixes in documentation codeblocks

### PR DESCRIPTION
**Pull request by NicolaiMogensen**
Several JSON code snippets in the documentation were not valid, i fixed them. Trailing commas on the last entry was the most common problem, which the JSON specification does not allow.

### CODE REVIEW

File: doc/python.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ results as `Numpy`_ arrays:
 
     json = """
     {
-      [
```
Comment (abellgithub) [inline doc/python.rst]: I think the intent of the original change was to remove the base object and "pipeline" key.

File: doc/stages/readers.tindex.rst [hunk 2]
```diff
@@ -43,7 +43,7 @@ merge the data.
           "type":"readers.tindex",
           "filter_srs":"+proj=lcc +lat_1=43 +lat_2=45.5 +lat_0=41.75 +lon_0=-120.5 +x_0=399999.9999999999 +y_0=0 +ellps=GRS80 +units=ft +no_defs",
           "filename":"index.sqlite",
-          "where":"location LIKE \'%nteresting.las%\'",
+          "where":"location LIKE \"%nteresting.las%\"",
```
Comment (abellgithub) [inline doc/stages/readers.tindex.rst]: I think the original was fine here.

Comment (NicolaiMogensen) [reply doc/stages/readers.tindex.rst]: I think so too, might have been my linter 5ec3f92

File: doc/python.rst (hunk 1 above)
Comment (NicolaiMogensen) [reply doc/python.rst]: Right, removing the opening and closing {} then as {[]} is not valid JSON 5ec3f92

**Comment by abellgithub** [comment]
Thanks!

**Pull request merged**